### ISSUE TITLE
Disqus comments blocked by Privacy Badger

### ISSUE THREAD

**Issue by anonsubmitter** [EFForg/privacybadgerfirefox#559]
Sample: https://torrentfreak.com/netflix-needs-bittorrent-expert-to-implement-p2p-streaming-150520/

Currently Disqus comments on all sites using Disqus are being blocked by Privacy Badger. Changing a.disquscdn.com from being completely blocked (red) to just blocking cookies from the domain (yellow) fixes this.

**Comment by jawz101** [comment]
I understand what you're saying but what you suggest to fix is what you'd do if you want to use them.  If they do not honor Do Not Track then that's what you'd have to do but it's a commenting system based on a tracking mechanism.  The whole concept is to make a tracker into a feature... which isn't great.  I just allow it temporarily if I am compelled to read or write comments but it's not as if I want it to follow me around everywhere Disqus is.

https://en.wikipedia.org/wiki/Disqus#Criticism_and_privacy_concerns

**Comment by rugk** [comment]
Duplicate of https://github.com/EFForg/privacybadgerfirefox/issues/469
